web/src/app/services/submission/submission.service.ts

```diff
@@ -86,7 +86,13 @@ export class SubmissionService {
             )
           )
         )
-        .subscribe(o => this.selectedSubmission$.next(o))
+        .subscribe(o => {
+          if (o instanceof Error) {
```

Comment:
You could also use ".filter(filterAndLogError)" above to avoid needing this in subscribe.

Reply:
Done.